core/embed/bootloader/header.S

```diff
@@ -10,7 +10,11 @@
 g_header:
     .byte 'T','R','Z','B'            // magic
     .word g_header_end - g_header    // hdrlen
+#ifdef TREZOR_MODEL_T
     .word 0                          // expiry
+#else
+    .word 1                          // expiry
```

Comment:
Why do we set this to 1 for model != T?

Reply:
This breaks compatibility of non TT images with older TT bootloaders. So that even already released bootloaders will recognize TR firmware as invalid.